File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/AsyncCommandsModule.cs
using MVVMDemo.AsyncCommands;
using DevExpress.Xpf.DemoBase;

namespace MVVMDemo {
    public partial class AsyncCommandsModule : MVVMDemoModule {
        protected override IShowcaseInfo[] CreateShowcases() {
            const string path = "Modules/AsyncCommands";
            const string uri = "17354/Common-Concepts/MVVM-Framework/Commands/Asynchronous-Commands";
            return new ShowcaseInfo[] {
                LoadShowcase("Asynchronous Delegate Commands", uri, path, new[] { typeof(AsyncDelegateCommandsViewModel), typeof(AsyncDelegateCommandsView) }),
                LoadShowcase("Asynchronous POCO Commands", uri, path, new[] { typeof(AsyncPOCOCommandsViewModel), typeof(AsyncPOCOCommandsView), typeof(AsyncPOCOCommandsViewModel_RuntimeGeneratedCode) }),
            };
        }
    }
}

File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/AsyncCommandsModule.vb
Imports MVVMDemo.AsyncCommands
Imports DevExpress.Xpf.DemoBase

Namespace MVVMDemo

    Public Partial Class AsyncCommandsModule
        Inherits MVVMDemoModule

        Protected Overrides Function CreateShowcases() As IShowcaseInfo()
            Const path As String = "Modules/AsyncCommands"
            Const uri As String = "17354/Common-Concepts/MVVM-Framework/Commands/Asynchronous-Commands"
            Return New ShowcaseInfo() {LoadShowcase("Asynchronous Delegate Commands", uri, path, {GetType(AsyncDelegateCommandsViewModel), GetType(AsyncDelegateCommandsView)}), LoadShowcase("Asynchronous POCO Commands", uri, path, {GetType(AsyncPOCOCommandsViewModel), GetType(AsyncPOCOCommandsView), GetType(AsyncPOCOCommandsViewModel_RuntimeGeneratedCode)})}
        End Function
    End Class
End Namespace
